src/sensai/vector_model.py

```diff
@@ -291,32 +213,16 @@ def fit(self, X: pd.DataFrame, Y: Optional[pd.DataFrame], fitPreprocessors=True,
             if Y is None:
                 raise Exception(f"The underlying model requires a data frame for fitting but Y=None was passed")
             X = self._computeModelInputs(X, Y=Y, fit=fitPreprocessors)
-            if self._targetTransformer is not None:
-                if fitTargetTransformer:
-                    Y = self._targetTransformer.fitApply(Y)
-                else:
-                    Y = self._targetTransformer.apply(Y)
             self._modelInputVariableNames = list(X.columns)
-            self._modelOutputVariableNames = list(Y.columns)
-            log.info(f"Training with outputs[{len(self._modelOutputVariableNames)}]={self._modelOutputVariableNames}, inputs[{len(self._modelInputVariableNames)}]=[{', '.join([n + '/' + X[n].dtype.name for n in self._modelInputVariableNames])}]")
+            log.info(
+                f"Training with outputs[{len(Y.columns)}]={list(Y.columns)}, inputs[{len(self._modelInputVariableNames)}]=[{', '.join([n + '/' + X[n].dtype.name for n in self._modelInputVariableNames])}]")
             self._fit(X, Y)
             self._isFitted = True
 
     @abstractmethod
     def _fit(self, X: pd.DataFrame, Y: pd.DataFrame):
         pass
 
-    def getModelOutputVariableNames(self):
```

Comment:
Look for references to this functions; you'll probably find a few dead ones in classification models.
I am on the fence as to whether this method should exist in VectorModel and return predictedVariablesNames or make this a breaking change and only fix it in sensai itself for the time being.

Reply:
I'll have a look within sensai. I would find this method confusing in the classifier's public interface, given that post-processing is gone there now

Reply:
Improvement #34 will remove the need for implementations to use any of these functions.

Reply:
I will implement it one of these days; I'd like to.
So for now, just change it to use predictedVarNames.

Reply:
Found one and fixed it. You have a good "Riecher", I didn't even think about that